index.html

```diff
@@ -9,6 +9,29 @@
         <link rel="stylesheet" href="main.css">
     </head>
     <body>
-        
+        <nav>
+            <p>Welcome</p>
+            <a href="#"><img src="img/Icon-Menu.svg" alt="Menu"></a>
+        </nav>
```

Comment:
- [ ]  Please consider removing the `nav` tag which you used to wrap the logo and the icon menu as we need the `nav` tag to only wrap the icon menu. Please replace it with a `header` tag as the logo and the menu are what make up the header/toolbar section.